src_c/mouse.c

```diff
@@ -390,11 +452,9 @@ static PyMethodDef _mouse_methods[] = {
     {"get_visible", mouse_get_visible, METH_NOARGS, DOC_PYGAMEMOUSEGETVISIBLE},
     {"get_focused", (PyCFunction)mouse_get_focused, METH_VARARGS,
      DOC_PYGAMEMOUSEGETFOCUSED},
-    {"set_cursor", mouse_set_cursor, METH_VARARGS, DOC_PYGAMEMOUSESETCURSOR},
-    {"set_system_cursor", mouse_set_system_cursor, METH_VARARGS, DOC_PYGAMEMOUSESETSYSTEMCURSOR},
-    {"get_cursor", (PyCFunction)mouse_get_cursor, METH_VARARGS,
-     DOC_PYGAMEMOUSEGETCURSOR},
-
+    {"set_system_cursor", mouse_set_system_cursor, METH_VARARGS, "set_system_cursor(constant) -> None\nset the mouse cursor to a system variant"},
```

Comment:
We should use the docs from the header files here, not inline ones. eg. DOC_PYGAMEMOUSESETCURSOR as before.

Reply:
The problem is that when the docs in `mouse.rst` are changed, the `setup.py docs` system regenerates the headers, and I removed set system cursor from the docs. Therefore regenerating the mouse docs causes the MOUSE_SETSYSTEMCURSOR constant to be deleted.

Reply:
Probably not removing set_system_cursor from the docs might be a better option?

Reply:
set system cursor has been publically available outside of prereleases for only two weeks. I think it should be removed from the docs to so that they more simply guide people towards the new way of doing cursors.

Reply:
Ok, thanks for the explanation. Seems it's good to leave it out of the docs.